src/components/WalletModal/Option.tsx

```diff
@@ -114,39 +154,65 @@ export default function Option({
   isActive?: boolean
   id: string
 }) {
+  const redesignFlag = useRedesignFlag()
+  const redesignFlagEnabled = redesignFlag === RedesignVariant.Enabled
+
   const content = (
     <TraceEvent
       events={[Event.onClick]}
       name={EventName.WALLET_SELECTED}
       properties={{ wallet_type: header }}
       element={ElementName.WALLET_TYPE_OPTION}
     >
-      <OptionCardClickable
-        id={id}
-        onClick={onClick}
-        clickable={clickable && !isActive}
-        active={isActive}
-        data-testid="wallet-modal-option"
-      >
-        <OptionCardLeft>
-          <HeaderText color={color}>
-            {isActive ? (
-              <CircleWrapper>
-                <GreenCircle>
-                  <div />
-                </GreenCircle>
-              </CircleWrapper>
-            ) : (
-              ''
-            )}
-            {header}
-          </HeaderText>
-          {subheader && <SubHeader>{subheader}</SubHeader>}
-        </OptionCardLeft>
-        <IconWrapper size={size}>
-          <img src={icon} alt={'Icon'} />
-        </IconWrapper>
-      </OptionCardClickable>
+      {redesignFlagEnabled ? (
+        <OptionCardClickable
+          id={id}
+          onClick={onClick}
+          clickable={clickable && !isActive}
+          active={isActive}
+          redesignFlag={true}
+          data-testid="wallet-modal-option"
+        >
+          <OptionCardLeft>
+            <HeaderText color={color} redesignFlag={true}>
+              <IconWrapper size={size}>
+                <img src={icon} alt={'Icon'} />
+              </IconWrapper>
+              {header}
+            </HeaderText>
+            {subheader && <SubHeader>{subheader}</SubHeader>}
+          </OptionCardLeft>
+          {isActive && <CheckIcon />}
+        </OptionCardClickable>
+      ) : (
+        <OptionCardClickable
+          id={id}
+          onClick={onClick}
+          clickable={clickable && !isActive}
+          active={isActive}
+          redesignFlag={redesignFlagEnabled}
+          data-testid="wallet-modal-option"
+        >
+          <OptionCardLeft>
+            <HeaderText color={color} redesignFlag={redesignFlagEnabled}>
+              {isActive ? (
+                <CircleWrapper>
+                  <GreenCircle>
+                    <div />
```

Comment:
is this needed?

Reply:
this is the old version of the code (pre redesign update) so prefer to leave as is?